### DISCUSSION TITLE
fhir.bundle.timestamp

### DISCUSSION THREAD

**Discussion by acekf01** [synthetichealth/synthea#1293] [General]
is there a way to generate a fhir bundle with the bundle.timestamp value (this is optional element in r4) ?
if not can a flag be created to allow this option ?

**Comment by jawalonoski** [comment]
Bottom-line: you'd have to edit the FHIR exporter, or post-process the data.

From https://hl7.org/fhir/R4/bundle-definitions.html#Bundle.timestamp

`Bundle.timestamp` is described as

> The date/time that the bundle was assembled - i.e. when the resources were placed in the bundle.

In the field comments it says this:

> transaction | transaction-response | batch | batch-response | collection : no particular assigned meaning

Not sure why this would be useful... would you expect the `instant` Synthea was executed, or the `instant` in which the synthetic provider would have created the record inside the simulation?

**Reply by acekf01** [reply]
I'm coming into a use case when ingesting 'fhir sample bundles' into an application, it is references to the bundle timestamp and is used to support 'instance' reconciliation (statefulness). (either case.. just need a timestamp)
